pipeline/vars/common.groovy

```diff
@@ -13,36 +13,16 @@ def prepareNode() {
     sh(script: "bash ${env.WORKSPACE}/pipeline/vars/node_bootstrap.bash")
 }
 
-def runTestSuite() {
+def getCLIArgsFromMessage(){
     /*
-        Execute the test suite by processing CI_MESSAGE for container
-        information along with the environment variables. The required
-        variables are
-            - sutVMConf
-            - sutConf
-            - testSuite
-            - addnArgs
+        Creates the CLI from CI Message
```

Comment:
Could we add more docStrings? `Returns the arguments after processing the CI message`

Reply:
Added the doc string.